FEXCore/Source/Interface/Core/ArchHelpers/Arm64Emitter.h

```diff
@@ -84,7 +84,8 @@ constexpr uint64_t EC_CODE_BITMAP_MAX_ADDRESS = 1ULL << 47;
 #endif
 
 // Will force one single instruction block to be generated first if set when entering the JIT filling SRA.
-constexpr auto ENTRY_FILL_SRA_SINGLE_INST_REG = TMP1;
+// FillStaticRegs must preserve this
+constexpr auto ENTRY_FILL_SRA_SINGLE_INST_REG = TMP2;
```

Comment:
https://github.com/FEX-Emu/FEX/blob/fe61aabc523ee5f48ead554ad14c37544f0223ba/Source/Windows/ARM64EC/Module.S#L52
The register here also needs to be changed right?

Reply:
Oh good catch